src/js/util/dataset-utils.ts

```diff
@@ -88,15 +90,16 @@ function output(id) {
  * @param   {Object} dataset  The definition for a dataset (e.g., name, url).
  * @returns {Object} An object containing the loaded values.
  */
-function loadURL(url) {
-  var fileName = url.match(NAME_REGEX)[1],
-      pipeline = Pipeline(fileName),
-      dataset  = Dataset(fileName, {url: url});
-
-  return promisify(dl.load)({url: url})
-    .then(function(data) {
-      return {data: data, pipeline: pipeline, dataset: dataset};
-    });
+function loadURL(url: string) {
+  const name = url.match(NAME_REGEX)[1];
+  const pipeline = Pipeline({name});
+  const dataset = Dataset(name, {url: url});
+
+  pipeline._id;
```

Comment:
does this line do anything?

Reply:
Hah, no. Good catch. I think I typed this when I was checking VSCode's autocomplete capabilities with the `Record` types.